### PR TITLE
add campaign_id to postcard and letter retrieval

### PR DESCRIPTION
**Pull request by shannamurry**
Fixes #\<INSERT ISSUE NUMBER HERE\>
GET request for /postcards/:id and /letters/:id now return campaign_id

## Checklist

- [ ] Up to date with `main`
- [ ] All the tests are passing
  - [ ] Delete all resources created in tests
- [ ] Prettier
- [ ] Spectral Lint
- [ ] `npm run bundle` outputs nothing suspect
- [ ] `npm run postman` outputs nothing suspect

### CODE REVIEW

File: resources/letters/models/letter_generated_base.yml [hunk 1]
```diff
@@ -29,3 +29,8 @@ allOf:
         enum:
           - letter
         default: letter
+
+      campaign_id:
```
Comment (sudoku-lord) [inline resources/letters/models/letter_generated_base.yml:33]: nitpick: could you put this above the object property? that generally should be the last one listed

File: resources/postcards/models/postcard.yml [hunk 2]
```diff
@@ -58,3 +58,8 @@ allOf:
         allOf:
           - description: A [signed link](#section/Asset-URLs) to the rendered postcard.
           - $ref: "../../../shared/attributes/signed_link.yml"
+
```
Comment (sudoku-lord) [inline resources/postcards/models/postcard.yml:61]: same thing as above, could you move url & campaign ID below object?

**Review by sudoku-lord — CHANGES_REQUESTED** [review]
Some minor changes and a point of clarification.

File: docs/chunks/styles.min.css [hunk 3]
```diff
@@ -1 +1 @@
...
...t;justify-content:start;margin-top:50px}.server-error .maintenance-text{margin-left:0;text-align:center}.server-error .maintenance-text h1{font-size:40px}.server-error .maintenance-image{margin-top:30px}}.navbar-sticky .w-nav-link{margin-top:3px}@media(max-width:991px){.navbar-sticky .w-nav-link{margin-top:0}.navbar-sticky .w-col{float:none}}#wrap{padding-top:95px}pre.prettyprint{display:block}pre .nocode{background-color:none;color:#000}pre .str{color:#416581}pre .kwd{color:#1878e0}pre .com{color:#27333c}pre .typ{color:#0099d7}pre .lit{color:#5f83a5}pre .pln,pre .pun{color:#324350}pre .atn,pre .atv,pre .dec,pre .tag{color:#1878e0}ol.linenums{color:#aeaeae;margin-bottom:0;margin-top:0}li.L0,li.L1,li.L2,li.L3,li.L5,li.L6,li.L7,li.L8{list-style-type:decimal!important}@media print{pre.prettyprint{background-color:none}code .str,pre .str{color:#adfac9}code .kwd,pre .kwd{color:#006;font-weight:700}code .com,pre .com{color:#600;font-style:italic}code .typ,pre .typ{color:#deb4ff;font-weight:700}code .lit,pre .lit{color:#044}code .pun,pre .pun{color:#440}code .pln,pre .pln{color:#000}code .tag,pre .tag{color:#006;font-weight:700}code .atn,pre .atn{color:#404}code .atv,pre .atv{color:#060}}
```
[the remark this answers is not in the document]
Comment (shannamurry) [reply docs/chunks/styles.min.css:1]: I think the linter is doing this automatically.  I manually added the new line back at the end but when I committed it deleted the new line - there is no change other then the new line being removed

**Pull request closed without merging**